statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1327,3 +1332,15 @@ class GLMResultsWrapper(lm.RegressionResultsWrapper):
     #                        exposure = data.exog[:,-1]).fit(tol=1e-12,
     #                                                        maxiter=250)
     #assert(np.all(res1.params == exposuremod.params))
+
+
+
+
+# Links that preserve the domain of the family.
+import statsmodels.genmod.families.links as lnk
+_domain_safe_links = {families.Binomial: (lnk.logit, lnk.probit, lnk.cauchy,),
+                      families.Poisson: (lnk.log,),
```

Comment:
I think this would be easier to maintain if it is in the family where currently the allowed links are defined.
eg. split
```
domain_safe_links = [...]
links = domain_safe_links + [L.log, L.identity]
```
or similar

for example Binomial is missing `clolog`, and when we add `loglog` then it only has to be added at one point to the family.